test/CMakeLists.txt

```diff
@@ -175,7 +175,7 @@ endforeach()
 # Add pytest
 # =============================================================================
 if(NRN_ENABLE_PYTHON AND PYTEST_FOUND)
-  set(pytest -m pytest)
+  set(pytest -m pytest --capture=tee-sys)
```

Comment:
Did you check this with a sanitizer error? I wonder if this manages to print errors that are immediately followed by a hard abort...

Reply:
did not test but it should work. I tested with #2024 where we could see the new dimension warning only by doing `pytest -s`

Reply:
I think the sanitizer errors are "harder" than that. It would be good to check.

Reply:
Well let's merge and see what gives?

Reply:
so `tee-sys` has `-s` included. Will test it today